app/javascript/dashboard/routes/dashboard/customviews/DeleteCustomViews.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <div>
+    <woot-delete-modal
+      v-if="showDeletePopup"
+      :show.sync="showDeletePopup"
+      :on-close="closeDeletePopup"
+      :on-confirm="deleteSavedCustomViews"
+      :title="$t('FILTER.CUSTOM_VIEWS.DELETE.MODAL.CONFIRM.TITLE')"
+      :message="deleteMessage"
+      :confirm-text="deleteConfirmText"
+      :reject-text="deleteRejectText"
+    />
+  </div>
+</template>
+
+<script>
+import alertMixin from 'shared/mixins/alertMixin';
+export default {
+  mixins: [alertMixin],
+  props: {
+    showDeletePopup: {
+      type: Boolean,
+      default: false,
+    },
+    activeCustomView: {
+      type: Array,
+      default: () => [],
+    },
+    customViewsId: {
+      type: [String, Number],
+      default: 0,
+    },
+  },
+
+  computed: {
+    deleteMessage() {
+      return `${this.$t(
+        'FILTER.CUSTOM_VIEWS.DELETE.MODAL.CONFIRM.MESSAGE'
+      )} ${this.activeCustomView[0] && this.activeCustomView[0].name} ?`;
+    },
+    deleteConfirmText() {
+      return `${this.$t('FILTER.CUSTOM_VIEWS.DELETE.MODAL.CONFIRM.YES')} ${this
+        .activeCustomView[0] && this.activeCustomView[0].name}`;
```

Comment:
Why are passing the prop `activeCustomView` as an array? It should be an object.

Reply:
FIxes by https://github.com/chatwoot/chatwoot/pull/3780/commits/d871b62e5a6aa6cba36f5f57616f85fa9a0a5adb